crates/turbopack-node/src/lib.rs

```diff
@@ -215,7 +217,18 @@ pub async fn get_renderer_pool(
         bail!("can only render from a disk filesystem, but `entrypoint = {}`", entrypoint.fs().to_string().await?);
     };
 
-    Ok(NodeJsPool::new(cwd, entrypoint, HashMap::new(), 4, debug).cell())
+    Ok(NodeJsPool::new(
+        cwd,
+        entrypoint,
+        env.read_all()
+            .await?
+            .iter()
+            .map(|(k, v)| (k.clone(), v.clone()))
+            .collect(),
+        4,
```

Comment:
This should probably not hard-code 4 child processes, but use `available_concurrency` instead

Reply:
done